src/cli-validator/utils/printResults.js

```diff
@@ -94,7 +99,13 @@ module.exports = function print(
   });
 
   // print the stats here, if applicable
-  if (reportingStats && (stats.errors.total || stats.warnings.total)) {
+  if (
+    reportingStats &&
+    (stats.errors.total ||
+      stats.warnings.total ||
+      stats.infos.total ||
+      stats.hints.total)
+  ) {
```

Comment:
I snuck this change into this PR.  It actually addresses [this comment](https://github.com/IBM/openapi-validator/pull/219/files/91760490fa119aa1199dd2f91837e5a1422f3ee5#r549391425) on PR #219.